tests/classification/_sklearn_calibration.py

```diff
@@ -0,0 +1,150 @@
+import numpy as np
```

Comment:
could you move the implementation to this file:
https://github.com/PyTorchLightning/metrics/blob/master/tests/helpers/non_sklearn_metrics.py